### PR TITLE
Feature #22768

### LINKED ISSUE #22768: Add clear recent projects from file menu to commands [ ctrl + shift + p ]
Add shortcut from (File -> Open Recent -> clear items) to commands  [ ctrl + shift + p ]

### PR DESCRIPTION
**Pull request by cristianhosu**
Fixes #22768

### CODE REVIEW

File: src/vs/code/electron-main/menus.ts [hunk 1]
```diff
@@ -511,7 +511,8 @@ export class VSCodeMenu {
 
 		if (folders.length || files.length) {
 			openRecentMenu.append(__separator__());
-			openRecentMenu.append(new MenuItem(this.likeAction('clearRecentlyOpened', { label: mnemonicLabel(nls.localize({ key: 'miClearItems', comment: ['&& denotes a mnemonic'] }, "&&Clear Items")), click: () => this.windowsService.clearRecentPathsList() }, false)));
+			//workbench.action.clearRecentItems
```
Comment (bpasero) [inline src/vs/code/electron-main/menus.ts]: Extra comment?

File: src/vs/workbench/browser/parts/editor/editorActions.ts [hunk 2]
```diff
@@ -1114,6 +1115,26 @@ export class ReopenClosedEditorAction extends Action {
 	}
 }
 
+export class ClearRecentItemsAction extends Action {
+
+	public static ID = 'workbench.action.clearRecentItems';
+	public static LABEL = nls.localize('clearRecentItems', "Clear Recent");
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/editor/editorActions.ts]: `Clear Recent Files` maybe?

File: src/vs/workbench/browser/parts/editor/editor.contribution.ts [hunk 3]
```diff
@@ -323,6 +323,7 @@ registry.registerWorkbenchAction(new SyncActionDescriptor(ShowEditorsInGroupThre
 registry.registerWorkbenchAction(new SyncActionDescriptor(OpenNextEditor, OpenNextEditor.ID, OpenNextEditor.LABEL, { primary: KeyMod.CtrlCmd | KeyCode.PageDown, mac: { primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.RightArrow, secondary: [KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.US_CLOSE_SQUARE_BRACKET] } }), 'View: Open Next Editor', category);
 registry.registerWorkbenchAction(new SyncActionDescriptor(OpenPreviousEditor, OpenPreviousEditor.ID, OpenPreviousEditor.LABEL, { primary: KeyMod.CtrlCmd | KeyCode.PageUp, mac: { primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.LeftArrow, secondary: [KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.US_OPEN_SQUARE_BRACKET] } }), 'View: Open Previous Editor', category);
 registry.registerWorkbenchAction(new SyncActionDescriptor(ReopenClosedEditorAction, ReopenClosedEditorAction.ID, ReopenClosedEditorAction.LABEL, { primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KEY_T }), 'View: Reopen Closed Editor', category);
+registry.registerWorkbenchAction(new SyncActionDescriptor(ClearRecentItemsAction, ClearRecentItemsAction.ID, ClearRecentItemsAction.LABEL), 'View: Clear Items', category);
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/editor/editor.contribution.ts]: `Clear Recent Files` maybe?

**Review by bpasero — CHANGES_REQUESTED** [review]

File: src/vs/code/electron-main/menus.ts (hunk 1 above)
Comment (cristianhosu) [inline src/vs/code/electron-main/menus.ts]: Sorry, i don't understand what you mean

Comment (mlewand) [inline src/vs/code/electron-main/menus.ts]: It means that you have left a commented out line `//workbench.action.clearRecentItems`, just remove it. 🙂

Comment (cristianhosu) [inline src/vs/code/electron-main/menus.ts]: Oh, i see. :) Sorry, forgot about that one.

**Comment by cristianhosu** [comment]
@bpasero - can you please tell me why is the build failing?

**Comment by bpasero** [comment]
LGTM, I think the test failures are unrelated 👍

**Pull request merged**